docs/README.md

```diff
@@ -0,0 +1,74 @@
+## Pygame Documentation Overview
+
+### Accessing Documentation
+
+Obviously you can visit pygame.org/docs to see the documentation, 
+but the documentation can also be launched with `python -m pygame.docs`
```

Comment:
This was meant to be more general than accessing docs you have built. And when you invoke it like this, with global pygame installed by pip, you always get the website version. Although it would be kind of cool to give people local copies of the documentation with default installations.

Reply:
Oh my bad, I didn't know we could launch this command with production version, nice! 👍